### DISCUSSION TITLE
Error in simulation while coupling with modflow6

### DISCUSSION THREAD

**Discussion by Abicula** [iiasa/CWatM#102] [Q&A] [answered]
Dear all,

As I am running CWatM coupled w/ modflow6, I received this error message:
"'ModFlowSimulation' object has no attribute 'head'"

![image](https://github.com/iiasa/CWatM/assets/123258196/95176470-7bcd-4168-9f81-f0e1a2af987a)

Please help,

**Comment by Abicula** [comment]
There is no need for further input, my team and I figured this out...
Apparently some packages need 'specific' version to run seamlessly, so make sure that all the packages are the same version as the "requirement.txt" file:

flopy==3.3.5
GDAL @ file:///C:/Users/smilovic/Downloads/GDAL-3.4.3-cp311-cp311-win_amd64.whl
netCDF4==1.6.1
numpy==1.23.4
openpyxl==3.0.10
pandas==1.5.1
rasterio @ file:///C:/Users/smilovic/Downloads/rasterio-1.2.10-cp311-cp311-win_amd64.whl
scipy==1.9.3
xlrd==1.2.0
xmipy==1.0.0

Python 3.8

Another note,
rasterio seems to work with me on version 1.3.9

![image](https://github.com/iiasa/CWatM/assets/123258196/171247b3-068d-4b05-ab6d-44cd24751b28)

**Comment by mikhailsmilovic** [comment, marked as answer]
The package xmipy==1.0.0 needs to be specific (related to Modflow). For example, with pip install
pip install xmipy==1.0.0

The other packages in their most recent versions should be fine.
